docs/source/running.rst

```diff
@@ -0,0 +1,20 @@
+.. _running:
+
+
+====================
+Running the Notebook
+====================
+
+After you have installed the Jupyter Notebook on your computer, you are ready
+to run the Notebook. To do this, run the command::
```

Comment:
Note you say it after, but I think that should be before, and probably not in a note.